cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/CassandraCluster.java

```diff
@@ -1320,4 +1326,36 @@ static long secondsUntilNextPossibleServerLaunch(final long now, final long next
         }
         return millisUntilNext / 1000;
     }
+
+    public void syncSeedsIfRequired() {
+        long lastTime = clusterState.get().getLastSeedFetchTime();
+        long now = System.currentTimeMillis();
```

Comment:
Please use `Clock` instead of `System.currentTimeMillis()` so that this code is more easily tested.